src/oop/object_interactions/medium/q3/MainTest.java

```diff
@@ -0,0 +1,170 @@
+    private final String getMyYearMethodName = "getMyYear";
+    private final String getMyMonthMethodName = "getMyMonth";
+    private final String getMyDayMethodName = "getMyDay";
+    private final String getMyStoryMethodName = "getMyStory";
+    private final String makeCopyMethodName = "makeCopy";
+
+    @BeforeEach
+    public void setup() throws Throwable {
+        String classString = "oop.object_interactions.medium.q3." + className;
+        String testClassString = "oop.object_interactions.medium.q3." + testClassName;
+        testClass = new ObjectTest(classString);
+        classObject = new ObjectTest(testClassString);
+        assertTrue(testClass.hasModifier("public"),
+                "You have changed the visibility modifier on the " + className + " class. Please revert it to the original state.");
+        assertTrue(testClass.hasField(myYearAttributeName, int.class, new String[]{"private"}),
+                String.format("You have modified the provided %s attribute. Please revert it to the original state.", myYearAttributeName));
+        assertTrue(testClass.hasField(myMonthAttributeName, int.class, new String[]{"private"}),
+                String.format("You have modified the provided %s attribute. Please revert it to the original state.", myMonthAttributeName));
+        assertTrue(testClass.hasField(myDayAttributeName, int.class, new String[]{"private"}),
+                String.format("You have modified the provided %s attribute. Please revert it to the original state.", myDayAttributeName));
+        assertTrue(testClass.hasField(myStoryAttributeName, String.class, new String[]{"private"}),
+                String.format("You have modified the provided %s attribute. Please revert it to the original state.", myStoryAttributeName));
+        assertTrue(testClass.hasConstructor(null, new String[]{"public"}),
+                "You have modified the provided constructor on the " + className + " class. Please revert it to the original state.");
+        Object classInstance = testClass.createInstance();
+        _assertEquals(-1, testClass.getFieldValue(classInstance, myYearAttributeName),
+                "You have modified the provided constructor on the " + className + " class. Please revert it to the original state.");
+        _assertEquals(-1, testClass.getFieldValue(classInstance, myMonthAttributeName),
+                "You have modified the provided constructor on the " + className + " class. Please revert it to the original state.");
+        _assertEquals(-1, testClass.getFieldValue(classInstance, myDayAttributeName),
+                "You have modified the provided constructor on the " + className + " class. Please revert it to the original state.");
+        _assertEquals("", testClass.getFieldValue(classInstance, myStoryAttributeName),
+                "You have modified the provided constructor on the " + className + " class. Please revert it to the original state.");
+
```

Comment:
None of these tests the method functions, they could all theoretically be returning 9999 and the test would pass.